### PR TITLE
Add support for specifying response file encoding

### PR DESCRIPTION
**Pull request by juj**
second attempt at landing #15406

### CODE REVIEW

File: ChangeLog.md [hunk 1]
```diff
@@ -59,6 +59,10 @@ See docs/process.md for more on how version tagging works.
 - Added SAFE_HEAP=2 option which tests safe heap behavior for wasm-only builds
   (allowing unaligned memory accesses, which would not work in Wasm2JS but in
    wasm would be correct but potentially slow).
+- Added support for specifying the text encoding to be used in response filenames
+  by passing the encoding as a file suffix (e.g. "a.rsp.utf-8" or "a.rsp.cp1252").
+  If not specified, the encoding is autodetected as either UTF-8 or Python
+  default "locale.getpreferredencoding()". (#15406, #15292, #15426)
```
Comment (sbc100) [inline ChangeLog.md:65]: Can you move this up 2.0.33 above?

**Review by sbc100 — APPROVED** [review]

**Comment by sbc100** [comment]
Can you title this that same as before ("Add support for specifying response file encoding") and mention in the commit message that it is a second attempt at landing #15406?

**Comment by sbc100** [comment]
Can you add this new test to `test-windows` in `.circleci/config.yml` ?

File: ChangeLog.md [hunk 2]
```diff
@@ -36,6 +36,10 @@ See docs/process.md for more on how version tagging works.
 2.0.33 - 11/01/2021
 -------------------
 - Bug fixes
+- Added support for specifying the text encoding to be used in response filenames
```
Comment (curiousdannii) [inline ChangeLog.md:39]: Changelog in wrong place?

**Pull request merged**